client/server/pages/test/index.js

```diff
@@ -1179,8 +1232,10 @@ describe( 'main app', () => {
 				'https://wordpress.com/wp-login.php?redirect_to=https%3A%2F%2Fwordpress.com%2Fplans'
 			);
 		} );
-
 		it( 'redirects to public pricing page', async () => {
+			app.withConfigEnabled( {
+				'jetpack-cloud/connect': false,
```

Comment:
Could you please elaborate on this one?

Reply:
It is a small drive-by improvement.

This tests is asserting the behaviour when `config.isEnabled( 'jetpack-cloud/connect' )` is false, but it was relying on the fact that the mocked `config.isEnabled()` returns `undefined` by default. I think being explicit about which config key is expected to be disabled makes the test more correct.

Reply:
Good one 👍